src/de/jost_net/JVerein/gui/view/EinstellungenAnzeigeView.java

```diff
@@ -69,6 +69,7 @@ public void bind() throws Exception
     left.addLabelPair("Mitgliedsfoto *", control.getMitgliedfoto());
     left.addLabelPair("Mittelverwendung anzeigen" + "*",
         control.getMittelverwendung());
+    left.addLabelPair("Projekte anzeigen" + "*", control.getProjekte());
```

Comment:
```suggestion
    left.addLabelPair("Projekte anzeigen *", control.getProjekte());
```

Konkatenation erscheint mir hier überflüssig. (Auch wenn wir es bei manchen Punkten so machen, aber eben nicht bei allen)